src/Components/Assets/AssetsList.tsx

```diff
@@ -105,9 +110,13 @@ const AssetsList = () => {
       facility_external_id: String(qParams.facility),
       external_id: String(qParams.location),
     },
-    prefetch: !!(qParams.facility && qParams.location),
+    prefetch: qParams.facility || qParams.location,
```

Comment:
```suggestion
    prefetch: !!(qParams.facility && qParams.location),
```

This will cause request to be dispatched with undefined right?

<img width="1822" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/ba35c58c-7149-47c1-a200-2ae7fd6c517d">

Reply:
Yeah, it wasn't working for me locally earlier....strange